gulpfile.js

```diff
@@ -433,8 +433,9 @@ function exportCommonJS(targetDir) {
 
 		var logManagerExportTask;
 		if (fileExists(PATHS.BUNDLEROOT + "logManager_internal.js") && !argv.nointernal) {
+			let ariaLibToInclude = argv.nominify ? "aria-web-telemetry-2.7.1" : "aria-web-telemetry-2.7.1.min.js"; 
```

Comment:
What would you think about pulling the version number out into a variable and then using that to build the file name?  It might make things a little cleaner going forward (updating, etc.).